package/lib/src/controls/text.dart

```diff
@@ -113,26 +114,30 @@ class TextControl extends StatelessWidget with FletStoreMixin {
               ? SelectableText.rich(
                   TextSpan(text: text, style: style, children: spans),
                   maxLines: maxLines,
+                  textDirection: rtl ? TextDirection.rtl : null,
```

Comment:
Instead of setting individual `textDirection` props I would wrap multiple content into [Directionality](https://api.flutter.dev/flutter/widgets/Directionality-class.html) class.

We could do `rtl` as a separate https://github.com/flet-dev/flet/issues/2462 - with behavior similar to `adaptive` property.